exposed-kotlin-datetime/src/test/kotlin/org/jetbrains/exposed/sql/kotlin/datetime/KotlinTimeTests.kt

```diff
@@ -469,28 +469,19 @@ open class KotlinTimeBaseTest : DatabaseTestsBase() {
                         .single()[testTable.timestampWithTimeZone.date()]
                 )
 
+                val expectedTime =
+                    when (testDb) {
+                        TestDB.SQLITE -> OffsetDateTime.parse("2023-05-04T05:04:01.123+00:00")
+                        TestDB.MYSQL_V8, TestDB.SQLSERVER,
+                        TestDB.H2_V2_ORACLE, TestDB.ORACLE,
+                        TestDB.H2_V2_PSQL, TestDB.POSTGRESQL, TestDB.POSTGRESQLNG ->
+                            OffsetDateTime.parse("2023-05-04T05:04:01.123123+00:00")
+                        else -> now
+                    }.toLocalTime().toKotlinLocalTime()
                 assertEquals(
-                    now.year,
-                    testTable.select(testTable.timestampWithTimeZone.year()).where { testTable.id eq nowId }
-                        .single()[testTable.timestampWithTimeZone.year()]
-                )
-
-                assertEquals(
-                    now.month.value,
-                    testTable.select(testTable.timestampWithTimeZone.month()).where { testTable.id eq nowId }
-                        .single()[testTable.timestampWithTimeZone.month()]
-                )
-
-                assertEquals(
-                    now.dayOfMonth,
-                    testTable.select(testTable.timestampWithTimeZone.day()).where { testTable.id eq nowId }
-                        .single()[testTable.timestampWithTimeZone.day()]
-                )
-
-                assertEquals(
-                    now.hour,
-                    testTable.select(testTable.timestampWithTimeZone.hour()).where { testTable.id eq nowId }
-                        .single()[testTable.timestampWithTimeZone.hour()]
```

Comment:
Did we want to remove all these asserts? They were left alone in the other module test.

Reply:
No, this was by mistake. Thanks for catching it!